frontstage_api/controllers/party_controller.py

```diff
@@ -35,3 +35,41 @@ def get_party_by_email(email):
 
     logger.debug('Successfully retrieved party')
     return json.loads(response.text)
+
+
+def verify_token(token):
+    logger.debug('Verifying token party')
+    url = app.config['RAS_PARTY_VERIFY_PASSWORD_TOKEN'].format(token)
+    response = request_handler('GET', url, auth=app.config['BASIC_AUTH'])
+
+    if response.status_code != 200:
+        logger.error('Failed to verify token')
+        raise ApiError(url, response.status_code)
+
+    logger.debug('Successfully verified token')
+    return json.loads(response.text)
+
+
+def reset_password_request(username):
```

Comment:
we should probably log when entering this function to stay consistent with the other functions in our controllers